sourced/ml/cmd/preprocess_source.py

```diff
@@ -0,0 +1,28 @@
+import os
+import logging
+from uuid import uuid4
+
+from sourced.ml.transformers import FieldsSelector, Moder, DzhigurdaFiles, ParquetSaver, \
+    create_uast_source
+from sourced.ml.utils.engine import pipeline_graph, pause
+
+
+@pause
+def preprocess_source_entry(args):
```

Comment:
The command is named "preprocess". Better to remove "source" then - but please add the docstring

Reply:
I think it is better to rename both to `preprocess_siva` and `preprocess_siva_entry` because we have `id2vec_preproc` and later there will be more preprocess steps.
Do you argee?

Reply:
How about `preprocess_uast`? The storage format can change in the future.

Reply:
`preprocess_repositories` than? because it is not about `uast` maybe we need only `content`without uasts.

Reply:
`preprocess_repos` then